package.json

```diff
@@ -95,13 +80,13 @@
     "webpack-dev-server": "^1.14.1"
   },
   "dependencies": {
+    "pdfobject": "2.0.201604172",
```

Comment:
@stefaniacardenas just headsup, runtime dependencies should not go under dev dependencies ;)